recipes/ganon/meta.yaml

```diff
@@ -18,6 +18,7 @@ source:
     
 build:
   number: 0
+  skip: True # [osx]
```

Comment:
osx was skipped due to the lack of gcc support and version of Clang (requires at least 7 and it's current on the version 4)